app/services/api/append_metadata.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Api
+  class AppendMetadata
+    include ServicePattern
+
+    def initialize(data)
+      @data = data
+    end
+
+    def call
+      {
+        data:,
```

Comment:
At some stage I think we'll need to introduce a versionable serialiser kind of thing here (rather than let active model/record decide what to include in the JSON). I don't think we are ready for that until the analysis has been done on which attributes and associations to include.

Reply:
agreed!